daffodil-test/src/test/resources/org/apache/daffodil/section17/calc_value_properties/computedLengthFields.tdml

```diff
@@ -0,0 +1,215 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="DFDLTutorialStylesheet.xsl"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<testSuite suiteName="Computing Length Fields"
+					 xmlns="http://www.ibm.com/xmlns/dfdl/testData"
+					 xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData"
+					 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+					 xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+					 xmlns:xs="http://www.w3.org/2001/XMLSchema"
+					 xmlns:ex="http://example.com">
+
+	<tdml:defineSchema name="s" useDefaultNamespace="false" elementFormDefault="unqualified"
+										 xmlns="http://www.w3.org/2001/XMLSchema">
+
+		<include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
+
+				<dfdl:format ref="ex:GeneralFormat"
```

Comment:
This indentation is not this far off. Something with the viewer.

Reply:
Looks like this file is using tab for indents instead of spaces.

Reply:
There are tabs in this file and that's what is causing the indentation to look wonky.  If you have no reason to keep the tabs, it would be good to reindent this file using the usual two spaces per indentation level.